.github/actions/templates/avm-validateModuleDeployment/action.yml

```diff
@@ -78,6 +78,7 @@ runs:
             'avm/res/azure-stack-hci/logical-network'            # Failing on resource deletion when trying to delete RBAC at subscription level
             'avm/res/azure-stack-hci/network-interface'          # Failing on resource deletion when trying to delete RBAC at subscription level
             'avm/res/azure-stack-hci/virtual-hard-disk'          # Failing on resource deletion when trying to delete RBAC at subscription level
+            'avm/res/azure-stack-hci/virtual-machine-instance'   # Failing on resource deletion when trying to delete RBAC at subscription level
```

Comment:
Sounds good. This requires manual action before merge. Just adding this comment as a placeholder to avoid the merge to go in as is after review 🙂